src/main/java/cms/gov/madie/measure/resources/TestCaseController.java

```diff
@@ -128,22 +156,49 @@ public ResponseEntity<String> deleteTestCases(
         String.join(", ", testCaseIds),
         measureId);
 
+    // Check locks for all test cases before deleting
+    for (String testCaseId : testCaseIds) {
+      checkTestCaseLock(testCaseId, principal.getName());
```

Comment:
Maybe we don't need this. I see something related to lock checking here- https://github.com/MeasureAuthoringTool/measure-service/blob/bc90ae111575f7807d127dc6423aa2a5a4bb0d67/src/main/java/cms/gov/madie/measure/services/TestCaseService.java#L429

Reply:
Done